earth_enterprise/rpms/shared/lib/key-value.sh

```diff
@@ -46,6 +46,8 @@ keyvalue_file_escape_key()
 }
 
 # Escapes a value to a string that can be stored in the key-value store.
+# Note: this function does not escape null characters because they are not
```

Comment:
Minor: I'd say, more specifically, e.g. `because current versions of BASH don't support null characters in variable values`, since null characters can appear in other places, such as being piped to command input, or from command output.